app/controllers/player_controller.rb

```diff
@@ -5,7 +5,7 @@ class PlayerController < ApplicationController
   def show
     # increment only when the player is embedded in other site...
     unless params.key?(:inside) && params[:inside] == '1'
-      @slide.increment(:embedded_view).increment(:total_view).save
+      @slide.increment!(:embedded_view).increment!(:total_view)
```

Comment:
__[RuboCop]__

Rails/SkipsModelValidations: Avoid using `increment!` because it skips validations.


<!-- Comment by [SideCI](https://sideci.com) -->